pageserve/static/style.css

```diff
@@ -139,7 +142,7 @@ textarea {
 }
 
 .page_content {
-  margin: 0 15%;
+  margin: 75px 15% 30px;
```

Comment:
Did you test this on mobile too? The top bar might wrap over and overlap content if this isn't big enough. You can use the device emulators in Chrome dev tools (F12 then Ctrl+Shift+M) to test.

Reply:
If it's not enough an easy fix might be to find the width where the header wraps then do
```css
@media screen and (max-width: width-in-px) {
  .page_content {
    margin-top: 150px;
  }
}
```

Reply:
Done e99c822, looks like https://screenshot.googleplex.com/H8LQfUb5ygn on small screens.